### PR TITLE
[FSDE-8075] Migrate Flag Icon component to vue documentation

### PR DESCRIPTION
**Pull request by soujanreddy**

### CODE REVIEW

**Review by evansito — APPROVED** [review]

**Review by evansito — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-211/14/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://78d54ffd0223be244edf249a13d59eb229fe8d2e-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-211-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-211/15/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://b9512a8c9827525894ed9d5cd63f5e211ab55114-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-211-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: src/documentation/pages/components/flag-icon/examples/_area.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<template lang="pug">
+  <ComponentExample title="Country/Region" id="area_lumen_centurylink" :tabs="exampleTabs">
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/flag-icon/examples/_area.vue:2]:
```suggestion
  <ComponentExample title="Country/Region" id="country_region" :tabs="exampleTabs">
```

File: src/documentation/pages/components/flag-icon/examples/_area.vue [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+<i class="chi-flag-icon icon-jp" aria-hidden="true"></i>
+<i class="chi-flag-icon icon-mx" aria-hidden="true"></i>
+<i class="chi-flag-icon icon-sg" aria-hidden="true"></i>
+<i class="chi-flag-icon icon-gb" aria-hidden="true"></i>
+<i class="chi-flag-icon icon-us" aria-hidden="true"></i>`
+      }
+    };
+  }
+})
+export default class SolidExample extends Vue {
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/flag-icon/examples/_area.vue:72]:
```suggestion
export default class CountryRegion extends Vue {
```

File: src/documentation/pages/components/flag-icon/examples/_sizes.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
+<template lang="pug">
+  <ComponentExample title="Sizes" id="sizes_lumen_centurylink" :tabs="exampleTabs">
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/flag-icon/examples/_sizes.vue:2]:
```suggestion
  <ComponentExample title="Sizes" id="sizes" :tabs="exampleTabs">
```

File: src/documentation/pages/components/flag-icon/examples/_sizes.vue [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        webcomponent: ``,
+        htmlblueprint: `<!-- -xs : 18x12px -->
+<i class="chi-flag-icon icon-us -xs" aria-hidden="true"></i>
+<!-- -sm : 24x16px -->
+<i class="chi-flag-icon icon-us -sm" aria-hidden="true"></i>`
+      }
+    };
+  }
+})
+export default class SizesExample extends Vue {
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/flag-icon/examples/_sizes.vue:53]:
```suggestion
export default class Sizes extends Vue {
```

File: src/documentation/pages/components/flag-icon/examples/index.vue [hunk 5]
```diff
@@ -0,0 +1,21 @@
+<template lang="pug">
+  div
+    h2 Examples
+    div
+      <Area />
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/flag-icon/examples/index.vue:5]:
```suggestion
      <CountryRegion />
```

File: src/documentation/pages/components/flag-icon/examples/index.vue [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+      <Sizes />     
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import Area from './_area.vue';
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/flag-icon/examples/index.vue:11]:
```suggestion
import CountryRegion from './_country_region.vue';
```

File: src/documentation/pages/components/flag-icon/examples/index.vue [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+import Sizes from './_sizes.vue';
+
+@Component({
+  components: {
+    Area,
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/flag-icon/examples/index.vue:16]:
```suggestion
    CountryRegion,
```

File: src/documentation/pages/components/flag-icon/examples/index.vue [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+    Sizes,
+  }
+})
+export default class FlagIconExamples extends Vue {}
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/flag-icon/examples/index.vue:20]:
```suggestion
export default class Examples extends Vue {}
```

File: src/documentation/pages/components/flag-icon/examples/_area.vue [hunk 9]
```diff
@@ -0,0 +1,75 @@
+<template lang="pug">
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/flag-icon/examples/_area.vue:1]: Change filename to **`_country_region.vue`**

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

**Review by josecarlosgonzalezv — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-211/16/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://f3e8358fd1cf6d76785e308d8d468aef21aeb65c-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-211-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-211/17/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://b04e9574bd01acb834176ce14eed700e676acbc3-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-211-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-211/18/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://f1b7d6970a70cf5b4bcc0697e464024ed72d72c4-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-211-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-211/19/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://d3641d11490297282e8ef80ebd051304e8a30d38-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-211-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-211/20/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://2a768d4669ed69c9636506be6ba27584cf5efea6-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-211-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-211/21/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://e5794e310dbbc5cf70183e1f376210efed0a78df-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-211-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by jllr — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-211/22/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-211/23/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-211/24/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-211/25/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://53436193e3a8c6757982643e98ad4bd68f80ff8c-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-211-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by janibolkvadze — APPROVED** [review]

**Pull request merged**